src/components/TextToSpeech.tsx

```diff
@@ -0,0 +1,55 @@
+import { FC, ReactElement, useEffect, useState, ReactNode } from "react";
+import { Box, Button, Typography, TypographyProps } from "@mui/material";
+import PlayArrowIcon from '@mui/icons-material/PlayArrow';
+import { SxProps, SystemProps } from '@mui/system';
+
+
```

Comment:
if you need your own props you can declare them here:
```suggestion
export interface TextToSpeechProps extends TypographyProps {
iconSize?: number;
} 
```

Reply:
decided to only use props passed down by typography